tools/pipelines-tasks/common/helpers.ts

```diff
@@ -116,4 +122,26 @@ export const writeXml = (xmlObject: any, filePath: string) =>
 {
     const xmlBuilder = new xml.Builder();
     fs.writeFileSync(filePath, xmlBuilder.buildObject(xmlObject));
+}
+
+export async function installNuget(packagePath: string, outputPath: string, targetNetFramework: string, packageId?: string, version?: string) {
```

Comment:
Some functions could benefit from more robust error handling. `installNuget` use await without a try/catch block, which could lead to unhandled promise rejections if an error occurs.

Reply:
Agreed, added exception handling for installNuget